docker-compose.debug.yml

```diff
@@ -1,42 +1,44 @@
 version: '3.8'
 services:
   webapp:
-    build: .
-    command: "npm run build:run"
+    build:
+      context: .
+      target: dev
     restart: unless-stopped
     ports:
       - "8080:8080"
     volumes:
-      - ".:/app"
-    depends_on:
-      - server
+      - "./src:/app/src"
+      - "./app.js:/app/app.js"
```

Comment:
This I already mentioned. These are the only source files you'll be editing that need hot reloading so it should suffice.